### PR TITLE
Add eye operator support

### PR DESCRIPTION
**Pull request by yeliang2258**
1. Add eye operator support

### CODE REVIEW

File: paddle2onnx/mapper/tensor/eye.cc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "paddle2onnx/mapper/tensor/eye.h"
+
+namespace paddle2onnx {
+REGISTER_MAPPER(eye, EyeMapper)
+
+int32_t EyeMapper::GetMinOpset(bool verbose) { return 9; }
```
Comment (jiangjiajun) [inline paddle2onnx/mapper/tensor/eye.cc]:
```
int32_t EyeMapper::GetMinOpset(bool verbose) {
    Logger(verbose, 9) << RequireOpset(9) << std::endl;
    return 9;
}
```

File: tests/run.sh [hunk 2]
```diff
@@ -74,7 +73,7 @@ bug=0
 export PY_CMD=$1
 $PY_CMD -m pip install pytest
```
Comment (jiangjiajun) [inline tests/run.sh:75]: eye单测中，增加num_columns为-1的测试情况

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

**Comment by jiangjiajun** [comment]
另外PR中不用再增加 `in cpp version`这种描述，就是增加`Add eye operator support`即可

File: tests/run.sh (hunk 2 above)
Comment (yeliang2258) [reply tests/run.sh:75]: 设置当num_columns为-1时，设置num_columns = num_rows

File: paddle2onnx/mapper/tensor/eye.cc (hunk 1 above)
Comment (yeliang2258) [reply paddle2onnx/mapper/tensor/eye.cc]: Done

**Pull request merged**